js/pages/characterizations/services/conversion/BaseStatConverter.js

```diff
@@ -76,10 +76,15 @@ define([
                 defaultColNames: this.getDefaultColumns().map(col => col.title),
                 perStrataColNames: this.getReportColumns(0,0).map(col => col.title),
                 columns: columns,
+                defaultSort: this.getDefaultSort(columns.length, cohorts.length),
                 data: Array.from(data.values()),
             };
         }
 
+        getDefaultSort(columnsCount, cohortsCount) {
+            return [[ columnsCount - 1, "desc" ]];
```

Comment:
Let's add comment that this is either for Std Diff (in comparison mode) or Pct (in single cohort mode)